src/target/source/codegen_cuda.cc

```diff
@@ -1051,18 +1051,27 @@ void CodeGenCUDA::PrintWmmaScope(const std::string& scope, DataType t, const Var
   }
 }
 
+int stoi(const std::string& str) {
+  try {
+    return std::stoi(str);
+  } catch (std::invalid_argument& e) {
+    LOG(FATAL) << "Cannot convert \"" << str << "\" to int";
+    throw;
```

Comment:
I don't think you need this throw, LOG(FATAL) throws under the hood.

Reply:
Somehow the compiler can't see through LOG(FATAL). I think it is because it actually throws in the destructor.